Toolset/palettes/tutorial/courses/First Run/tutorials/Combining Text/lessons/App Lesson.txt

```diff
@@ -0,0 +1,270 @@
+tutorial "Combining Text"
+
+prologue
+	LiveCode's uniquely powerful text handling capabilities comes in handy for a wide range of applications. Common tasks include:
+
+	- Mail merging a form email
+
+	- Creating a leaflet for print
+
+	- Building an invoice
+end prologue
+
+step "Tutorial Intro 1"
+	In this tutorial, you'll learn how to:
+
+	- Work with words and lines
+
+	- Combine texts together
+
+	- Make decisions
+
+	- Insert variables into your output
+action
+	interlude
+	go to step "Load stack"
+end step
+
+step "Load stack"
+	Some text
+action
+	load lesson "App Lesson 0"
+	go to step "Tutorial Intro 2"
+end step
+
+step "Tutorial Intro 2"
+	In this tutorial, we will have a template email, which is placed in a field called "template", a record, which is placed in a field called "record", and an output field, which is called "output". We also have a button called "Create email".
+action
+	interlude
+	go to step "Address by name 1"
+end step
+
+step "Address by name 1"
+	We want to create a personalized email by combining the email template with the personal details in the "record" field.
+action
+	interlude
+	go to step "Address by name 2"
+end step
+
+step "Address by name 2"
+	The first step is to start our email with "Dear Mr Miller". To do this we need to add some code to the button.
+
+	Ensure you are in 'Edit' mode and select the "Create email" button .
+action
+	wait until button "create" is selected
+	go to step "Address by name 3"
+end step
```

Comment:
You should make the edit mode bit a separate step. It will likely already be in edit mode and if this ends up being the first tutorial someone does somehow then it will be confusing- how do I go into edit mode?